example_workflow/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: pymser
+channels:
+  - conda-forge
+dependencies:
+  - python >= 3.10
+  - pip
+  - numpy
+  - pandas
+  - gemmi
+  - pip:
+    - pymser
+    - RASPA2
```

Comment:
For some reason it did not work on my computer:

> Installing initially failed dependencies...

It could be just a matter of configure pipenv (I've never used it).